unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/ChannelMembersComponentView.cs

```diff
@@ -83,11 +89,22 @@ public void ShowLoading()
 
         public void Set(ChannelMemberEntryModel user) => queuedEntries.Enqueue(user);
 
+        public void Remove(string userId)
+        {
+            memberList.Remove(userId);
+            UpdateLayout();
+            UpdateHeaders();
```

Comment:
Shouldn't you update the headers first and then set the layout dirty with the updatelayout?

Reply:
`UpdateHeaders()` is to updating the text of the header, however `UpdateLayout()` is only to update the scroll transform... they are different sections of the UI so the order of these functions doesn't matter.